core/src/main/java/org/apache/accumulo/core/spi/balancer/SimpleLoadBalancer.java

```diff
@@ -265,7 +265,10 @@ List<TabletMigration> move(ServerCounts tooMuch, ServerCounts tooLittle, int cou
         log.error("Unable to select a tablet to move", ex);
         return result;
       }
-      TabletId tabletId = selectTablet(onlineTabletsForTable);
+      if (onlineTabletsForTable.isEmpty()) {
+        return result;
+      }
+      TabletId tabletId = onlineTabletsForTable.keySet().iterator().next();
```

Comment:
Can probably remove the `if (tabletId == null) { return result;}` a few lines down.  The new `if (onlineTabletsForTable.isEmpty()) {` a few lines above probably covers the case it was covering.

Reply:
Removed the other `if` in b82d5e3